dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetrySourcesTest.java

```diff
@@ -0,0 +1,289 @@
+        String traceLine = server.waitForStringInLog("finest trace", 5000, server.getConsoleLogFile());
+        String ffdcLine = server.waitForStringInLog("liberty_ffdc", 5000, server.getConsoleLogFile());
+
+        Map<String, String> attributeMap = new HashMap<String, String>() {
+            {
+                put("io.openliberty.ext.app_name", "MpTelemetryLogApp");
+                put("io.openliberty.type", "liberty_message");
+                put("io.openliberty.module", "io.openliberty.microprofile.telemetry.logging.internal.fat.MpTelemetryLogApp.MpTelemetryServlet");
+                put("thread.id", "");
+                put("thread.name", "");
+                put("io.openliberty.sequence", "");
+            }
+        };
+
+        assertNotNull("Info message log could not be found.", messageLine);
+        assertTrue("MPTelemetry did not log the correct message", messageLine.contains("info message"));
+        assertTrue("MPTelemetry did not log the correct message", messageLine.contains("INFO"));
+        checkJsonMessage(messageLine, attributeMap);
+
+        assertNull("Trace message was found and should not have been bridged.", traceLine);
+        assertNull("FFDC message was found and should not have been bridged.", traceLine);
+
+    }
+
+    /*
+     * Test a server with an empty MPTelemetry sources attribute and ensure no logs are bridged.
+     */
+    @Test
+    @ExpectedFFDC({ "java.lang.NullPointerException" })
+    public void testTelemetryEmptySource() throws Exception {
+        RemoteFile consoleLogFile = server.getConsoleLogFile();
+        setConfig(SERVER_XML_EMPTY_SOURCE, consoleLogFile, server);
+
+        TestUtils.runApp(server, "logServlet");
+        TestUtils.runApp(server, "ffdc1");
+
+        String messageLine = server.waitForStringInLog("info message", 5000, server.getConsoleLogFile());
+        String traceLine = server.waitForStringInLog("finest trace", 5000, server.getConsoleLogFile());
+        String ffdcLine = server.waitForStringInLog("liberty_ffdc", 5000, server.getConsoleLogFile());
+
+        // checkJsonMessage(line, messageKeyList, messageValueList);
+
+        assertNull("Info message was found and should not have been bridged.", messageLine);
+        assertNull("Trace message was found and should not have been bridged.", traceLine);
+        assertNull("FFDC message was found and should not have been bridged.", traceLine);
+
+    }
+
+    /*
+     * Test a server with an invalid MPTelemetry source attribute and a warning is logged.
+     */
+    @Test
+    public void testTelemetryInvalidSource() throws Exception {
+        RemoteFile consoleLogFile = server.getConsoleLogFile();
+        setConfig(SERVER_XML_INVALID_SOURCE, consoleLogFile, server);
+
+        TestUtils.runApp(server, "logServlet");
+
+        String messageLine = server.waitForStringInLog("CWMOT5005W", server.getDefaultLogFile());
+
+        assertNotNull("Unknown log source warning was not found.", messageLine);
```

Comment:
Ensure, no sources are bridged over as well, when an invalid source is configured. Might be good to also add a test, if we have an invalid source along with valid sources and ensure that the only valid sources are bridged over.